spec/fixtures/elasticsearch-flat-format.yml

```diff
@@ -0,0 +1,11 @@
+elasticsearch.host: http://test:9200
```

Comment:
Is there an example of the crawl.yml file in flat format too? Probably worth testing both, especially considering the two configs are merged post-nesting.